app/controllers/histogram_chart_controller.rb

```diff
@@ -48,13 +50,20 @@ def new
 		        end
 		        nd = (temp+ @freq).to_s
 		        st = temp.to_s
-		       	titles[ind] = (st + " to " + nd)
-		        @histoAr[ind] = freqNum
-		        temp += @freq
+		        title = (st + " to " + nd)
+            titles[ind] = title
```

Comment:
these titles are confusing, perhaps we should refactor and use better variable names.